tests/pathfinding/test_token_network.py

```diff
@@ -167,10 +167,26 @@ def test_check_path_request_errors(token_network_model, addresses):
         "Source does not have a channel with sufficient capacity"
     )
     token_network_model.G.edges[a[0], a[1]]["view"].capacity = 100
+
     assert token_network_model.check_path_request_errors(a[0], a[2], 100, reachability).startswith(
         "Target does not have a channel with sufficient capacity"
     )
     token_network_model.G.edges[a[1], a[2]]["view"].capacity = 100
 
     # Must return `None` when no errors could be found
-    assert token_network_model.check_path_request_errors(a[0], a[2], 100, reachability) is None
+    assert token_network_model.check_path_request_errors(a[0], a[2], 100, reachability,) is None
```

Comment:
Where does the trailing comma come from?

Reply:
Before I added another parameter and that required reformatting, and then I forgot to remove it.